app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+    workStack: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    name: 'Profesional Art Printing Data More',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    workStack: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    name: 'Data Dashboard Healthcare',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    workStack: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    name: 'Website Portfolio',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    workStack: ['HTML', 'Bootstrap', 'Rub000y'],
+  },
+];
+
+if (portfolio) {
+  const div = document.createElement('div');
+  const img = document.createElement('img');
+  div.classList.add();
+  projectsWrapperDiv.classList.add('recent-project-wraper', 'project-block');
+  mostRecentProjectWrapper.classList.add('most-recent-project-wraper');
+  descktopProjectsWrapperDiv.classList.add(
+    'desktop-recent-project-wraper',
+    'project-block',
+  );
+
+  img.setAttribute('src', `${recentProjects[0].image}`);
+  img.setAttribute('alt', 'recent work');
+  img.classList.add('recent-image');
+  div.classList.add('most-recent-work-project-details-waraper');
+  div.innerHTML = `<h3 class="recent">${recentProjects[0].name}</h3>
+            <p class="recent-project-details">
+              ${recentProjects[0].description}
+            </p>
+            <ul class="recent-buttons-container">
+              <li>
+                <a
+                  href="https://www.w3.org/Style/CSS/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-1"
+                  >CSS</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-2"
+                  >HTML</a
+                >
+              </li>
+              <li>
+                <a
+                  href="
+ https://getbootstrap.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-3"
+                  >Bootstrap</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-4"
+                  >Ruby</a
+                >
+              </li>
+            </ul>
+            <a
+              href="#0"
+              class="btn-see-project active-btn-effect hover-btn-effect pop"
+            >
+              See project
+            </a>`;
```

Comment:
👏👏 Great job on implementing the popup functionality! However, there are a couple of improvements that I'd like to suggest:

- [ ] Currently, the popup only works for the first project button, but as per the project requirement, it should work for each project button. You need to ensure that clicking on any "Show Project" button will display the relevant details for that specific project.

- [ ] Additionally, the content of the popup should be dynamically generated based on the specific project details. Currently, the popup shows static data, but it should display the title, description, technologies, and links related to the specific project that the user clicks on. To achieve this, make use of the project array object that you have and dynamically showcase the project details in the popup when the corresponding button is clicked.

By making these adjustments, your popup functionality will be more interactive and user-friendly, allowing users to view specific project details when they click on the respective buttons. Keep up the good work, and happy coding! 🚀💻🌟